src/main/scala/za/co/absa/spark/commons/version/SparkCompatibility.scala

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
I don't think this makes sense in the library. At least not in this format - maybe, maybe with #7.

Reply:
So then both classes wouldn't make sense?

Reply:
Let's talk about it 😉